ui/src/components/Publication/Creation/MainText.tsx

```diff
@@ -102,173 +212,164 @@ const MainText: React.FC = (): React.ReactElement | null => {
         updateReferences(references.filter((item) => item.id !== id));
     };
 
-    const handleClose = useCallback(() => {
-        setSelectedReference(null);
-    }, []);
-
-    const saveReferenceChanges = useCallback(
-        (reference: Interfaces.Reference) => {
-            const modifiedReference = getTransformedReference(reference);
-            updateReferences(
-                references.map((reference) => (reference.id === modifiedReference.id ? modifiedReference : reference))
-            );
-            setSelectedReference(null);
-        },
-        [references, updateReferences]
-    );
-
     return (
-        <div className="space-y-12 2xl:space-y-16">
-            <div>
-                <Components.PublicationCreationStepTitle text="Main text" required />
-                {publicationId && (
-                    <Components.TextEditor
-                        defaultContent={content}
-                        contentChangeHandler={updateContent}
-                        references={references}
-                    />
-                )}
-            </div>
+        <>
```

Comment:
You need to rebase this branch with the feature branch. I think you can also remove the react fragments as they are no longer necessary.